package.json

```diff
@@ -45,6 +45,7 @@
     "gulp": "^3.8.11",
     "mochawait": "^2.0.0",
     "proxyquire": "^1.6.0",
+    "sleep": "^3.0.1",
```

Comment:
Both `asyncbox` and `bluebird` have "sleep" functionality. To wait for a second:

```js
import B from 'bluebird';

await B.delay(1000);
```

```js
import { sleep } from 'asyncbox';

await sleep(1000);
```